src/components/Form/Form.css

```diff
@@ -8,30 +8,30 @@
   display: flex;
   flex-direction: column;
   justify-content: left;
-  position: sticky;
-  position: absolute;
+  position: fixed;
   top: 50%;
   left: 50%;
   margin-top: -263.5px;
   margin-left: -192px;
-  z-index: 2;
+  z-index: 1001;
```

Comment:
we already chatted about this, but we should either explore making the google map index 1 instead and pull this out of this pr OR we can just push like this for now and then make a backlog issue in trello to update later if time allows. i think that's ok since this is a stylistic nice to have, not must have

Reply:
i'm realizing that was for the overlay, which was even higher z-index. was the maps covering this form as well?

(again, non-blocking comment)

Reply:
The map was above the overlay. I'm not sure whether it was also above the form. But the form needs to be above the overlay